RPerformanceTracking/Private/_RPTClassManipulator+UIWebView.m

```diff
@@ -148,7 +148,7 @@ + (void)_swizzleUIWebViewDelegate:(id<UIWebViewDelegate>)delegate
             uint_fast64_t trackingIdentifier = [objc_getAssociatedObject(webView, _RPT_UIWEBVIEW_TRACKINGIDENTIFIER) unsignedLongLongValue];
             if (trackingIdentifier)
             {
-                [_RPTTrackingManager.sharedInstance.tracker end:trackingIdentifier statusCode:httpResponse.statusCode];
+                [_RPTTrackingManager.sharedInstance.tracker updateStatusCode:httpResponse.statusCode trackingIdentifier:trackingIdentifier];
```

Comment:
We only update the statusCode for didFinishLoad.
I think we should also do it for didFailLoad if we can get access to the httpurlresponse.
Then that common logic can be extracted into a helper method.